src/App.js

```diff
@@ -23,12 +24,11 @@ const App = () => {
             the Smart Shopping List!
           </p>
           <h2>Week 1</h2>
-          <ShoppingList />
-          <MenuLink activeWhenExact={true} to="/" label="List View" />
+          <MenuLink activeWhenExact={true} to="/ListView" label="List View" />
           <MenuLink to="/AddItem" label="Add Item" />
 
           <Switch>
-            <Route exact path="/">
+            <Route exact path="/ListView">
```

Comment:
Not specifically related to the AC for this issue but I noticed that this path name was changed here and not changed in the "Not Found" component when redirecting to the list view

Reply:
I think this was confusing because "Home" didn't exist previously so it was interchangable with /ListView as our intended default depending on the user's entry point. But I think with your Home component refactor and redirect handler we've tied these routes up nicely.